### PR TITLE
Pull Request From Admin Dashboard

### PR DESCRIPTION
**Pull request by DomingoLenar**
- Contains Admin Dashboard (index)

### CODE REVIEW

File: server/nodejs/resources/js/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ document.addEventListener('DOMContentLoaded', () => {
     const surveyData = [
         {
             survey_id: 1,
-            survey_title: 'Sample Survey 1',
+            survey_title: 'Sample Survey 1  ',
```
Comment (VladTemp27) [inline server/nodejs/resources/js/index.js:5]: Why does this have a space?

File: server/nodejs/resources/js/index.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ document.addEventListener('DOMContentLoaded', () => {
...
             status: 'unpublished',
             program_id: 101,
             period_start: '2024-11-01',
```
Comment (VladTemp27) [inline server/nodejs/resources/js/index.js:8]: Where is the time start here?

**Review by VladTemp27 — CHANGES_REQUESTED** [review]
Certain elements are missing such as time start, on top of questionable string values

File: server/nodejs/resources/js/index.js (hunk 1 above)
Comment (DomingoLenar) [reply server/nodejs/resources/js/index.js:5]: my bad, maybe i left it for testing purposes on ui

File: server/nodejs/resources/js/index.js (hunk 2 above)
Comment (DomingoLenar) [reply server/nodejs/resources/js/index.js:8]: i'll have to confirm this with geb, as far as we've talked about, this json was within the requirements, but we'll resolve this immediately.

File: server/nodejs/resources/css/admin-publish-survey.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    src: url("../../../php/public/resources/fonts/Istok_Web/IstokWeb-Regular.ttf") format("truetype");
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: "CustomFont";
+    src: url("../../../php/public/resources/fonts/Istok_Web/IstokWeb-Bold.ttf") format("truetype");
+    font-weight: bold;
+    font-style: normal;
+}
```
Comment (VladTemp27) [inline server/nodejs/resources/css/admin-publish-survey.css:12]: Please use the font in style.css this is redundant

**Review by VladTemp27 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**